src/main/java/de/tum/in/www1/artemis/web/rest/lecture/VideoUnitResource.java

```diff
@@ -92,7 +92,7 @@ public VideoUnitResource(LectureRepository lectureRepository, AuthorizationCheck
             new URL(videoUnit.getSource());
         }
         catch (MalformedURLException exception) {
-            badRequest();
+            return badRequest();
```

Comment:
Can we directly replace these calls?

Reply:
sure, you can, but at some point, I wanted to stop and prevent making this PR too big.
In fact, this was a bug before, because the previous code basically did nothing :-(